xbmc/video/VideoDatabase.cpp

```diff
@@ -394,7 +394,7 @@ void CVideoDatabase::CreateViews()
                                        "        episode.idShow=tvshow.idShow"
                                        "      LEFT JOIN files ON"
                                        "        files.idFile=episode.idFile "
-                                       "    GROUP BY tvshow.idShow");
+                                       "GROUP BY tvshow.idShow");
```

Comment:
Is this change necessary or just a leftover?

Reply:
Hi, not actually. But I practically need the final file after all changes as it is here to be the one to be tested: 2a90ad7

Reply:
If you can remove all the previous commits somehow and just leave the last image of the file it would be even better :-) I don't know how to do this myself...still new at this GIT system...

Reply:
Hey, we will need you to condense this into one commit before merging anyway :-)

It's called "squashing". You can find a simple explanation here: https://github.com/todotxt/todo.txt-android/wiki/Squash-All-Commits-Related-to-a-Single-Issue-into-a-Single-Commit
In this simple case it would also be possible that you just start fresh (reset the branch to master) and make a new commit from there. You then have to force-push (`git push --force`) because otherwise Git will not allow you to rewrite history.

Don't be afraid to ask if you have further questions.

Reply:
Thanks for your response. So please kindly confirm my understanding that I need to issue the below commands from what I read if I use the second option you listed:

1. git reset --hard feature_myvideos113 master        to reset the feature_myvideos113 branch to master
2. git add .     to add the change 
3. git commit -m "final commit"      to create the commit 
3. git push --force  origin feature_myvideos113   to re-push the final videodatabase.cpp file that I have locally in my branch as part of the new commit

Thanks and regards

Reply:
For 1: You first have to checkout the branch with `git checkout feature_myvideos113` if you are not already using it (which I assume). Then just do `git reset master`. If you use `git reset --hard`, it will also reset your files to the master branch and discard all of your changes. Without `--hard`, it just changes the branch pointer and you can then commit the changes again. Rest of the steps seems ok.

Reply:
Many thanks for the advice, I'll try those now...

Reply:
this will require a bumb of db version right? is it for v18?

Reply:
Uh why did you not remove the whitespace change here?

Reply:
let me check the file again
It seems the space is not there in my file, take a look:

![image](https://user-images.githubusercontent.com/23155784/50650203-7edea480-0f88-11e9-8889-ad3bccc7ee5b.png)

Reply:
From what I see from other select statements that have group by and I did not change,the identation used is that GROUP BY should follow immediately after the "
This is how I put it in my file to keep the identation consistent.

Reply:
Oh, you are of course absolutely right. Disregard :-)

Reply:
No worries, my big fear is whether the season_view will work or not....I hope I used the variables correctly, if I understood it correctly from reading the other select statements...this is why I need to test this...I am not sure why whoever chose to write this file used variables to include in the prepare statement....this is a level of abstraction that I am not used to...

Reply:
did it again and bumped the database version, please kindly check.